moveit_core/robot_model/src/planar_joint_model.cpp

```diff
@@ -139,39 +141,128 @@ void PlanarJointModel::getVariableRandomPositionsNearBy(random_numbers::RandomNu
   normalizeRotation(values);
 }
 
+void computeTurnDriveTurnGeometry(const double* from, const double* to, const double min_translational_distance,
```

Comment:
I'm not a fan of double* type. Would prefer a std::vector. Maybe @jliukkonen or @tylerjw has a strong opinion

I see you inherited the previous style, though

Reply:
This is a consequence of the interpolate signature in the base class where double* is used.